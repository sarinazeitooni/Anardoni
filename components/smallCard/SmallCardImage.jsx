import React from "react";
import style from './scss/smallCard.module.scss'
const SmallCardImage = ({cardImage})=>{
    return(
        <div className={style.cardWithoutContainerImage}>
            <img src={cardImage} width={95} height={95}/>
        </div>
    )
}
export default SmallCardImage;