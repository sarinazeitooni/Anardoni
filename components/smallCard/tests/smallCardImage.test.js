import React from 'react';
import { shallow } from 'enzyme';
import SmallCardImage from '../SmallCardDetail';
describe("SmallCardImage component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<SmallCardImage cardImage={"test"}/>);
        expect(container).toMatchSnapshot();
    });

});