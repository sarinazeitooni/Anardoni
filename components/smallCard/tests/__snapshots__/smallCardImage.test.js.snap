// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallCardImage component it should render a div 1`] = `ShallowWrapper {}`;
